src/digitalIdentity-backend/src/test/java/didentity/amos/digitalIdentity/repository/UserRepositoryTest.java

```diff
@@ -1,4 +1,39 @@
 package didentity.amos.digitalIdentity.repository;
 
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import didentity.amos.digitalIdentity.model.User;
+
 public class UserRepositoryTest {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @AfterEach
+    void tearDown() {
+        userRepository.deleteAll();
+    }
+
+    // test DataBase creation
+    @Test
+    void itShouldFindACreatedUser() {
```

Comment:
[ERROR]   DigitalIdentityApplicationTests.contextLoads » IllegalState Failed to load App...
[ERROR]   UserRepositoryTest.itShouldFindACreatedUser:30 NullPointer

Two tests are failing. Are there configurations on my side missing? Or do you have a clue why this happens

Reply:
I think the userRepository is not initialized there